pkg/terminal/command.go

```diff
@@ -561,6 +567,11 @@ func goroutines(t *Term, ctx callContext, argstr string) error {
 			prefix = "* "
 		}
 		fmt.Printf("%sGoroutine %s\n", prefix, formatGoroutine(g, fgl))
+		if printStack == true {
+			if _, err := t.client.SwitchGoroutine(g.ID); err == nil {
+				stackCommand(t, ctx, "")
```

Comment:
Replace this if with:

```
	stack, err := t.client.Stacktrace(g.ID, 10, nil)
	if err != nil {
		return err
	}
	printStack(stack, "\t", false)
```

That way the stack trace is indented and we don't call `SwitchGoroutine` which changes the state of the program.